pkg/cloudprovider/gce/gce.go

```diff
@@ -309,6 +309,7 @@ func translateAffinityType(affinityType api.AffinityType) GCEAffinityType {
 // CreateTCPLoadBalancer is an implementation of TCPLoadBalancer.CreateTCPLoadBalancer.
 // TODO(a-robinson): Don't just ignore specified IP addresses. Check if they're
 // owned by the project and available to be used, and use them if they are.
+// TODO: Verify this is idempotent
```

Comment:
Oh - I thought that was what each check to ignore StatusConflict was doing?

I'll add a item to my todo list for an e2e test that requires changing a loadbalancer (I think adding another port ought to do it)